Assets/Scripts/Models/Area/Room.cs

```diff
@@ -15,546 +15,360 @@
-            foreach (Tile tile2 in neighbours)
+            foreach (Tile tile in exits)
             {
-                if (tile2 == null || tile2.Room == null)
+                // Loop over the exits to find a different room
+                Tile[] neighbours = tile.GetNeighbours();
+                foreach (Tile neighbor in neighbours)
                 {
-                    continue;
-                }
+                    if (neighbor == null || neighbor.Room == null)
+                    {
+                        continue;
+                    }
 
-                // We have found a room
-                if (tile2.Room != this)
-                {
-                    neighboursRooms[tile2] = tile2.Room;
+                    // We have found a room
+                    if (neighbor.Room != this)
+                    {
+                        neighboursRooms[neighbor] = neighbor.Room;
+                    }
                 }
             }
-        }
 
-        return neighboursRooms;
-    }
-
-    // Changes gas by an amount in preasure(in atm) multiplyed by number of tiles
-    public void ChangeGas(string name, float amount)
-    {
-        if (IsOutsideRoom())
-        {
-            return;
+            return neighboursRooms;
         }
 
-        if (atmosphericGasses.ContainsKey(name))
+        // Changes gas by an amount in preasure(in atm) multiplyed by number of tiles
+        public void ChangeGas(string name, float amount)
         {
-            atmosphericGasses[name] += amount;
-            if (amount.IsZero()) 
+            if (IsOutsideRoom())
             {
-                deltaGas[name] = "=";
+                return;
             }
-            else if (Mathf.Sign(amount) == 1)
+
+            if (atmosphericGasses.ContainsKey(name))
             {
-                deltaGas[name] = "+";
+                atmosphericGasses[name] += amount;
+                if (amount.IsZero())
+                {
+                    deltaGas[name] = "=";
+                }
+                else if (Mathf.Sign(amount) == 1)
+                {
+                    deltaGas[name] = "+";
+                }
+                else
+                {
+                    deltaGas[name] = "-";
+                }
             }
             else
             {
-                deltaGas[name] = "-";
+                atmosphericGasses[name] = amount;
+                deltaGas[name] = "=";
             }
-        }
-        else
-        {
-            atmosphericGasses[name] = amount;
-            deltaGas[name] = "=";
-        }
 
-        if (atmosphericGasses[name] < 0)
-        {
-            atmosphericGasses[name] = 0;
+            if (atmosphericGasses[name] < 0)
+            {
+                atmosphericGasses[name] = 0;
+            }
         }
-    }
 
-    public string ChangedGases(string name)
-    {
-        if (deltaGas.ContainsKey(name))
+        public string ChangedGases(string name)
```

Comment:
This is not a method I changed, what happened was a bit of a whitespace change perhaps combined with a different source or something? I think git is just being weird. However I will change the name, just as a cleanup.